.github/workflows/power9.yml

```diff
@@ -0,0 +1,118 @@
+
+      - name: checkout_kokkos
+        uses: actions/checkout@v3
+        with:
+          repository: kokkos/kokkos
+          ref: ${{ github.base_ref }}
+          path: kokkos
+
+      - name: configure_kokkos
+        run: |
+          mkdir -p kokkos/{build-${{ matrix.backend }},install-${{ matrix.backend }}}
+          cd kokkos/build-${{ matrix.backend }}
+          GPP=$(which g++)
+          cmake -DCMAKE_CXX_COMPILER=$GPP \
+            -DCMAKE_CXX_FLAGS="-O3 -Wall -Wunused-parameter -Wshadow -pedantic -Werror -Wsign-compare -Wtype-limits -Wignored-qualifiers -Wempty-body -Wclobbered -Wuninitialized" \
+            -DCMAKE_EXE_LINKER_FLAGS= \
+            -DCMAKE_INSTALL_PREFIX=$PWD/../install-${{ matrix.backend }} \
+            -DKokkos_ENABLE_SERIAL=${{ matrix.serial }} \
+            -DKokkos_ENABLE_OPENMP=${{ matrix.openmp }} \
+            -DKokkos_ARCH_PASCAL60=ON \
+            -DKokkos_ARCH_POWER8=ON \
+            -DKokkos_ENABLE_TESTS=OFF \
+            -DKokkos_ENABLE_EXAMPLES=OFF \
+            -DCMAKE_VERBOSE_MAKEFILE=ON \
+            -DCMAKE_CXX_EXTENSIONS=OFF \
+            -DCMAKE_CXX_STANDARD=17 \
+            -DBUILD_SHARED_LIBS=OFF \
+            -DKokkos_ENABLE_DEPRECATION_WARNINGS=OFF \
+            -DKokkos_ENABLE_DEPRECATED_CODE_4=OFF \
+            -DKokkos_ENABLE_DEPRECATION_WARNINGS=OFF \
+            ..
+
+      - name: build_and_install_kokkos
+        working-directory: kokkos/build
+        run: make -j12 install
+
+      - name: configure_kokkos_kernels
+        run: |
+          mkdir -p kokkos-kernels/{build,install}-${{ matrix.backend }}
+          cd kokkos-kernels/build-${{ matrix.backend }}
+          GPP=$(which g++)
+          cmake \
+            -DCMAKE_CXX_COMPILER=$GPP \
+            -DKokkos_DIR=$PWD/../../kokkos/install-${{ matrix.backend }}/lib64/cmake/Kokkos \
+            -DCMAKE_CXX_FLAGS="-O3 -Wall -Wunused-parameter -Wshadow -pedantic -Werror -Wsign-compare -Wtype-limits -Wignored-qualifiers -Wempty-body -Wclobbered -Wuninitialized" \
+            -DCMAKE_INSTALL_PREFIX= \
+            -DKokkosKernels_ENABLE_TESTS_AND_PERFSUITE=OFF \
+            -DKokkosKernels_ENABLE_TESTS=ON \
+            -DKokkosKernels_ENABLE_PERFTESTS=ON \
+            -DKokkosKernels_ENABLE_EXAMPLES:BOOL=ON \
+            -DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=OFF \
+            -DKokkosKernels_INST_COMPLEX_DOUBLE=ON \
+            -DKokkosKernels_INST_DOUBLE=ON \
+            -DKokkosKernels_INST_ORDINAL_INT=ON \
+            -DKokkosKernels_INST_OFFSET_SIZE_T=ON \
+            -DKokkosKernels_INST_OFFSET_INT=ON \
+            -DKokkosKernels_INST_LAYOUTLEFT=ON \
+            -DKokkosKernels_ENABLE_TPL_ROCSPARSE=OFF \
+            -DKokkosKernels_ENABLE_TPL_ROCBLAS=OFF \
+            -DKokkosKernels_ENABLE_TPL_CUSOLVER=OFF \
+            -DKokkosKernels_ENABLE_TPL_BLAS=OFF \
+            -DKokkosKernels_ENABLE_TPL_CUBLAS=OFF \
+            -DKokkosKernels_ENABLE_TPL_CUSPARSE=OFF \
```

Comment:
Again all the cuda and rocm TPLs will be off automatically because the associated Kokkos backends are not enabled.

Reply:
same as above WRT cm_test_all_sandia.